README.md

```diff
@@ -45,6 +46,9 @@ provider=digitalocean region=... tag_name=... api_token=...
 # Google Cloud
 provider=gce project_name=... zone_pattern=eu-west-* tag_value=consul credentials_file=...
 
+# Kubernetes
+provider=kubernetes namespace=... label_key=role label_value=consul
+
```

Comment:
Note that namespace is optional; it will default to the `default` namespace in Kubernetes.